src/torchmetrics/functional/classification/exact_match.py

```diff
@@ -232,7 +232,7 @@ def exact_match(
     if task == "multiclass":
         assert num_classes is not None
         return multiclass_exact_match(preds, target, num_classes, multidim_average, ignore_index, validate_args)
-    if task == "multilalbe":
+    if task == "multilabel":
```

Comment:
@Borda As these conditions are present plenty in our classification codebase, don't we wanna replace them with `Enum` to avoid possible similar typos in the future?

Reply:
yes, that would be much better :rocket: 
do you want to take it or could you create an issue and share it with the community?

Reply:
created a new issue for tracking this:
https://github.com/Lightning-AI/metrics/issues/1478